src/core/utils/index.js

```diff
@@ -174,3 +174,9 @@ export const redirectToUrl = event => {
     document.body.removeChild(anchor)
   } catch (e) {}
 }
+
+export const takeSubmitEventData = (event) => {
+  const { landingId } = event.detail || {}
+
+  return { response_id: landingId }
```

Comment:
Please comment here once you find out if [landingId is the same as responseId](https://typeform.slack.com/archives/CSZC3BMMH/p1588002748098400)

Reply:
we need centralised place for this :D 
but @pchiwan said they wanted to introduce `response_id` in `stakhanov`, it's in their backlog. so the issue will be solved as soon as they have it

Reply:
If this is in their backlog (and it is possible to retrieve `response_id` in stakhanov) I would wait for that.